tests/e2e/testcases/network/stop/suite.go

```diff
@@ -14,9 +14,9 @@ import (
 	"github.com/onsi/gomega"
 )
 
-var _ = ginkgo.Describe("[Local Network]", ginkgo.Ordered, func() {
+var _ = ginkgo.Describe("[Local Network] Stop", ginkgo.Ordered, func() {
 	ginkgo.AfterEach(func() {
-		commands.CleanNetwork()
+		_, _ = commands.CleanNetwork()
```

Comment:
I should add here a test case for stopping a locally deployed L1. it should both stop the network and
the L1s

Reply:
added.